sky/utils/common_utils.py

```diff
@@ -12,7 +12,7 @@
 import sys
 import time
 import uuid
-from typing import Any
+from typing import Any, Optional
```

Comment:
nit: Why don't we change the order? `from typing ...` should precede `import uuid`.

Reply:
Good catch! Fixed. Thanks!